lib/cache/OperationWriter.js

```diff
@@ -7,13 +7,33 @@ const LRU = require('lru-cache');
 const {config, util: {BedrockError}} = require('bedrock');
 const {LruCache} = require('@digitalbazaar/lru-memoize');
 
-const PROMISE_QUEUE = new LruCache({max: 10, disposeOnSettle: true});
+/* Note: Max size is 1. This is to promise queue is only used for the update
+   count function. */
+const lruCacheOptions = {max: 1, disposeOnSettle: true};
+const UPDATE_COUNT_PROMISE_QUEUE = new LruCache(lruCacheOptions);
```

Comment:
A size of 1 doesn't make sense because there may be multiple ledger nodes serviced by a single process -- and queue entries would be constantly evicted if just two were receiving ops.